arduino/libify.sh

```diff
@@ -162,7 +173,8 @@ copy_file src/sys include/sys/tree.hxx
 
 copy_file src/utils src/utils/*.{cxx,hxx,c,h}
 
-rm -f ${TARGET_LIB_DIR}/src/utils/ReflashBootloader.cxx
+rm -f ${TARGET_LIB_DIR}/src/utils/ReflashBootloader.cxx \
+    ${TARGET_LIB_DIR}/src/utils/if_tcp_test_helper.hxx
```

Comment:
we have a big bunch of other _test_helpers.hxx, do you want to exclude those too?

Reply:
I was thinking of doing that just haven't had a chance to track all of them down yet. I'm out of town this week but will be home next Tues July 16) and can go over the list.

Reply:
@balazsracz added commits below to remove the helper bits that I could easily identify (test in filename and not referenced from cxx/hxx files outside of tests)